### DISCUSSION TITLE
RTS 1 SENEGAL

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#772] [Q&A]
### Channel Name

rts1

### Channel ID

RTS1.sn

### Website

https://www.rts.sn/tv/RTS_1#

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
I had a fix for this one, sadly since the host is down I never had the chance to implement it back again.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/sn.m3u#L42-L43
